src/Elmish.WPF/Binding.fs

```diff
@@ -1263,6 +1263,22 @@ type Binding private () =
     >> Binding.mapModel get
     >> Binding.mapMsgWithModel set
 
+  /// <summary>Creates a two-way binding.</summary>
```

Comment:
FYI I went with 1 empty line between the obsolete overload and the non-obsolete one and kept the 2 lines between methods.